.eslintrc

```diff
@@ -8,6 +8,12 @@
     "prettier/react"
   ],
   "plugins": ["jsx-a11y", "header"],
+  "env": {
```

Comment:
Can you clarify what exactly each change here does and why this was suddenly necessary now?

Because some of these envs are already set, others only make sense in a specific context.

Reply:
Yeah, I see. It was necessary because of a `no-undef` ESLint Error while using `fetch` globally via `jest-fetch-mock`. It includes the environments that expose globals when it is defined (e.g. `global.fetch` and `window.fetch`). What do you think?

Reply:
I'll be using `global.fetch` instead 🥂

Reply:
Works for me! Thanks!